tools/run_satwnds/satwnds_new_subset_template.yaml

```diff
@@ -72,6 +72,11 @@ observations:
         - name: "ncols_conf"
           size: variables/percent_confidence.ncols
 
+      globals:
+        - name: "bufr_version@MetaData"
```

Comment:
There is no need to use the `@MetaData` suffix on an attribute name. The `@MetaData` syntax is for variable names, and the purpose of the suffix is to denote which group the variable belongs to. You might want to consider dropping the `@MetaData` suffix so it's a little easier to read the attribute name.

Reply:
@srherbener,  I updated the code to remove @MetaData and renamed bufr_version with bufr_seq_version to accommodate @rmclaren's suggestion for clarity.
Also, I updated the string values, so they look exactly like in most WMO documents, thus easier to Google search for them. 

No more changes are anticipated for a while so merging would be fine. thanks!

Reply:
Thanks for making this change!

Reply:
Thanks for letting me know that you anticipate no more changes. I still need one more approval before merging.